### DISCUSSION TITLE
[Feature] Анимация клина, sprint_start и sprint_end

### DISCUSSION THREAD

**Discussion by KillMurphy** [OGSR/OGSR-Engine#290] [Ideas]
В последнем билде добавили поддержку анимаций неполной перезарядки — и это очень радует, но мне по прежнему не хватает анимаций jamming и misfire, как в том же GUNSLINGER. А то немного тупо, что у ГГ оружие просто заклинило, а нужно полностью перезаряжать оружие. Ещё, конечно, есть такая хотелка как sprint_start и sprint_end, ибо интерполяция между idle и idle_sprint работает порой кривовато или резко, но опять же, это не так важно как анимация клина

**Comment by joye-ramone** [comment]
Есть рабочие модели с этими анимациями ?

**Comment by KillMurphy** [comment]
Есть, но не у меня. Если надо, могу прислать, но позже

**Comment by molodoy-chelovek** [comment]
Ещё было бы неплохо сделать что-то с idle_moving во время передвижения в положении сидя. Скорость передвижения сидя и скорость анимации idle_moving не сочитаются друг с другом.

**Comment by Sh0tri** [comment]
> Есть рабочие модели с этими анимациями ?

Могу попробовать сделать эти анимации, если надо. Не прям супер-круто, но для теста сойдет

**Comment by Olil-byte** [comment]
По поводу клина, у меня есть своя реализация:
https://github.com/Olil-byte/OGSR-Engine/commit/4d8547f879f0cf54ef6a2b0920ec14f65f0af902
https://github.com/Olil-byte/OGSR-Engine/commit/874749e386386b5c49e3889e110ebb448dcdfc04
https://github.com/Olil-byte/OGSR-Engine/commit/8926fe8bec788c7f087e7c135668596076c882e5

Её демонстрация:
https://www.youtube.com/watch?v=u8dK7NkhbXM

**Comment by xrSimpAI** [comment]
Ну, неплохо

**Comment by xrSimpAI** [comment]
@Olil-byte кст, тебе ещё надо будет решить #179 , если ты серьезно будешь заниматься этой фичей в дальнейшем.

**Comment by Olil-byte** [comment]
> @Olil-byte кст, тебе ещё надо будет решить #179 , если ты серьезно будешь заниматься этой фичей в дальнейшем.
@KRodinn
https://github.com/Olil-byte/OGSR-Engine/commit/a0ddd9925718e2217acefed66a885b5b734d3bb4

**Comment by xrSimpAI** [comment]
Ну так сейвы не совместимы будут, скорее всего

**Comment by Olil-byte** [comment]
> Ну так сейвы не совместимы будут, скорее всего

ну а как решать проблему нет_пакетов?

**Comment by xrSimpAI** [comment]
Ну как-то. Я-то это точно не буду решать, мне это не нужно.

**Comment by xrSimpAI** [comment]
И ещё, [**>>тут<<**](https://www.amk-team.ru/forum/topic/10538-x-ray-extensions/page/54/?tab=comments#comment-983349) по слову **заклин** 4 бага. Не факт, что это все. Собственно поэтому я вообще не хочу в это лезть. Это ж всё фиксить придётся, наверняка ещё какая-нибудь хрень повылезает. В итоге этими анимациями заклинов создам себе больше проблем чем пользы.
Тем более что это особо никому не нужно, и анимации никто делать особо не будет.
